lib/routes/home/home_page.dart

```diff
@@ -34,17 +34,18 @@ class HomeState extends State<Home> with AutoLockMixin, HandlerContextProvider {
   final GlobalKey firstPaymentItemKey = GlobalKey();
   final ScrollController scrollController = ScrollController();
   final handlers = <Handler>[];
+  late final InputHandler inputHandler = InputHandler(
```

Comment:
This will break the lifecycle again.

I think `SendOptionsBottomSheet` should create its own `InputHandler` dealing with the lifecycle there.

A possibility is a duplication of event handling as there will be 2 or more InputHandlers. We should think of a mechanism to avoid that.

Reply:
> This will break the lifecycle again.
> 
> I think `SendOptionsBottomSheet` should create its own `InputHandler` dealing with the lifecycle there.
> 
> A possibility is a duplication of event handling as there will be 2 or more InputHandlers. We should think of a mechanism to avoid that.

#  ⚠️  Do not merge until lifecycle problem is addressed. ⚠️

Reply:
Can we not add the inputhandler to the _injector.dart_? @erdemyerebasmaz